src/python/pants/backend/docker/target_types.py

```diff
@@ -169,21 +169,30 @@ class DockerBuildSecretsOptionField(
     AsyncFieldMixin, DockerBuildOptionFieldMixin, DictStringToStringField
 ):
     alias = "secrets"
-    help = "Secret file to expose to the build (only if BuildKit enabled).\n\n" + dedent(
-        """\
-        Example:
-
-            docker_image(
-                secrets={
-                    "mysecret": "/local/secret",
-                }
-            )
-        """
+    help = (
+        "Secret file to expose to the build (only if BuildKit enabled).\n\n"
```

Comment:
```suggestion
        "Secret files to expose to the build (only if BuildKit enabled).\n\n"
```

Reply:
Arrg. Right, that line was a copy-paste from docker cli output, didn't pay enough attention to it :P 

OK to punt to a later PR, as it's a minor typo?

Reply:
Yeah not a big deal